pkgs/stdenv/cross/default.nix

```diff
@@ -66,7 +66,7 @@ in lib.init bootStages ++ [
              (hostPlatform.isLinux && !buildPlatform.isLinux)
              [ buildPackages.patchelf ]
         ++ lib.optional
-             (let f = p: !p.isx86 || p.libc == "musl" || p.libc == "wasilibc" || p.isiOS; in f hostPlatform && !(f buildPlatform))
+             (let f = p: !p.isx86 || p.libc == "musl" || p.libc == "wasilibc" || p.isiOS || p.isWindows; in f hostPlatform && !(f buildPlatform))
```

Comment:
I was wondering if we should just set this to `stdenv.hostPlatform != stdenv.buildPlatform`. It very rarely hurts anything (there could be some weird custom gnuconfig changes that break) to set it, we mostly just need it for weird platforms though.

Reply:
Oh! I think I remembered some of the logic was to avoid adding it twice, if native already has it.

We should just do a mass rebuild so everywhere sets it, always.

Reply:
I don’t think it’s necessary for native. Besides costing a mass rebuild, it also could break some custom scripts.

Reply:
@matthewbauer We should make a `dontAutoreconf` for that.

Reply:
I'd prefer to not have an option for this. Status quo is probably okay. You can always add the hook yourself.

Reply:
Sorry, I meant `dontUpdateAutotoolsGnuConfigScripts`. I we'll only need to add it in a few places.